### ISSUE TITLE
SQL Binding - Function name isn't applied to FunctionName attribute

### ISSUE THREAD

**Issue by Charles-Gagnon** [microsoft/azuredatastudio#19469]
1. Go through the create AF w/ SQL Binding flow
2. When prompted to enter the function name enter in some custom value
3. Finish creating the function

Results : 

The function is created, but only the class has the name entered earlier. The `FunctionName` attribute does not

![image](https://user-images.githubusercontent.com/28519865/169591699-5d4b9788-e6e5-45b2-9e5d-812d2dfed65d.png)

We should be updating that value too

**Comment by VasuBhog** [comment]
Closed via https://github.com/microsoft/sqltoolsservice/pull/1521

**Comment by Charles-Gagnon** [comment]
@VasuBhog How exactly does the linked PR solve this issue? That has nothing to do with updating the FunctionName attribute

**Comment by VasuBhog** [comment]
@Charles-Gagnon so I tested Azure Function with SQL Binding and found that it updates the functionName to the one set as the objectName. Therefore, the only potential PR improvements that I believed could have solved this was STS related changes. There were no edits on the [templates side](https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config) so this was the reason.

**Comment by Charles-Gagnon** [comment]
Looks like it was indeed fixed, but the STS changes should have nothing to do with that. Prompting for the name and then passing that to the createFunction call doesn't involve STS at all.

Instead it looks like this was because in https://github.com/Microsoft/azuredatastudio/commit/c86085352e3394298c2b4c35b263750797bfd8b5#diff-7f64cfb2eb79ca96d04076610bfa52d74063fe2951484cbdfe60155a2d6b695aL174 you added the functionName parameter (as you indicated needed to be done in this issue) to the createFunction call.

**Comment by VasuBhog** [comment]
I am quite sure this is not to do with our logic then it must have been updated on the Azure Functions side. The PR you linked dates back to end of April and the issue above was found in end of May. So, it is fixed and I did include it FunctionName parameter to be passed in properly much earlier. So I assume the issue was fixed from Azure Function side to use the functionName given to the CreateFunction to also be used as the class name and function name now.
